doc/USER_GUIDE.md

```diff
@@ -107,6 +108,14 @@ aws s3 ls s3://amazon-eks/1.23.9/2022-07-27/bin/linux/x86_64/
 
 To build using the example binaries above:
 ```bash
+# Using IMDS v2
+make k8s \
+  kubernetes_version=1.23.9 \
+  kubernetes_build_date=2022-07-27 \
+  arch=x86_64 \
+  imds_support='v2.0'
```

Comment:
I don't think we need these examples, the variable doc table should be sufficient.